controllers/Sitemap.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable func-names */
+/* eslint-disable no-restricted-syntax */
```

Comment:
Should not disable ESLint restriction. Remove these lines and follow ESLint recommendation.